### DISCUSSION TITLE
integrate fwupd with mint updater!

### DISCUSSION THREAD

**Discussion by remixedcat** [fwupd/fwupd#6484] [Ideas]
I think it would be very cool and very useful to integrate fwupd into the mint updater that way you'd have one place to manage all system updates! has this been discussed on other sites as well and have you discussed this with LVFS team?

 I know that EFI/BIOS updates are happening a lot more frequently these days due to the constant popping up of EFI exploits and ppl on mint don't have an easy way to keep on top of things. This would increase security and increase mint linux's popularity too!

**Comment by hughsie** [comment]
I think you need to file this against the linux mint update program. It's probably quite easy for them to query the list of updates from fwupd, and to deploy the update on the system as it's just a few DBus calls. It's something, however, that needs to be done by them, not us... :)

**Reply by remixedcat** [reply]
ok wil do! Thanks for the quick response!

**Reply by remixedcat** [reply]
posted on mint linux forums https://forums.linuxmint.com/viewtopic.php?t=409069 hope they see that!

**Comment by remixedcat** [comment]
https://github.com/linuxmint/mintupdate/issues/237 I linked this here too
